catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -251,7 +251,7 @@ public void create() {
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 		Sprite copy = cloneMe.cloneForCloneBrick();
-		copy.resetSprite();
+		//copy.resetSprite(); // can I safely remove this?
```

Comment:
I guess not, see https://github.com/Catrobat/Catroid/pull/1972#discussion_r83835195

But please test the clone feature with a Sprite containing a physics brick, e.g. SetBounceBrick.
It's important that the cloned sprite also executes such a brick _after_ it has been cloned. I guess a [RuntimeException("No physics action available in non-physics sprite!");](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java#L764) will be thrown.